CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## [v0.0.26](http://github.com/ndustrialio/contxt-sdk-js/tree/v0.0.26) (2018-xx-xx)
```

Comment:
I just realized this. We might be doing [SemVer](https://semver.org/) wrong. This might qualify as a patch—no new features, bug fixes/documentation fixes—but new features should be bumping the minor version.

Reply:
We're not following SemVer yet :D. Was trying to get a roadmap out to see where we wanted to be for a 1.0 release.